src/runtime_src/xdp/profile/plugin/aie_profile/aie_plugin.cpp

```diff
@@ -687,7 +734,7 @@ namespace xdp {
     auto it = mThreadCtrlMap.find(handle);
     if (it == mThreadCtrlMap.end())
       return;
-
+    
```

Comment:
Please configure your editor to avoid adding random useless spaces.

Reply:
fixed